xep-0440.xml

```diff
@@ -71,35 +79,35 @@
 
 <section1 topic='Announcing the SASL Channel-Binding Type Capability' anchor='sasl-cb-type'>
 
-  <p>This protocol consists of a single optional extension element
-  named 'sasl-channel-binding' qualified by the 'urn:xmpp:sasl-cb:0'
-  namespace. The 'sasl-channel-binding' element MUST contain one or
+  <p>This protocol consists of a stream-feature named 'sasl-channel-binding'
```

Comment:
nit: remove the hyphen in stream feature. neither the RFCs nor most XEPs use a hyphen here.
```suggestion
  <p>This protocol consists of a stream feature named 'sasl-channel-binding'
```